pkg/xen/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN [ -f "$(basename ${XEN_SOURCE})" ] || tar --absolute-names -xz < /xen.tar.gz
 WORKDIR /xen/xen
 COPY *.patch arch /tmp/
 RUN cp /tmp/"$(uname -m)"/*.patch /tmp/
-RUN for p in /tmp/*.patch ; do patch -p1 < "$p" || exit 1 ; done
+#RUN for p in /tmp/*.patch ; do patch -p1 < "$p" || exit 1 ; done
```

Comment:
Should we instead delete the content of the patch directories? Or will we need some of them?

Reply:
We still need them, but it's just a temporary workaround to avoid applying the not-yet-ported patches.

Reply:
(it should be converted to a draft PR)